### PR TITLE
Dismissed

### PR DESCRIPTION
**Pull request by andrewdyer**

### CODE REVIEW

File: js/src/admin/components/DismissedFlagsTable.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    return (
+      <div className="DismissedFlagsTable">
+        <table>
+          <thead>
+            <td>{app.translator.trans('flarum-flags.admin.dismissed.fields.username')}</td>
+            <td>{app.translator.trans('flarum-flags.admin.dismissed.fields.discussion')}</td>
+            <td>{app.translator.trans('flarum-flags.admin.dismissed.fields.reason')}</td>
+            <td>{app.translator.trans('flarum-flags.admin.dismissed.fields.reason_detail')}</td>
+            <td>{app.translator.trans('flarum-flags.admin.dismissed.fields.created_at')}</td>
+            <td>{app.translator.trans('flarum-flags.admin.dismissed.fields.dismissed_at')}</td>
```
Comment (imorland) [inline js/src/admin/components/DismissedFlagsTable.js]: Shouldn't we have the 'dismissed by' user in here too?

File: js/src/admin/components/DismissedFlagsPage.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// import sortable from 'html5sortable/dist/html5sortable.es.js';
```
Comment (imorland) [inline js/src/admin/components/DismissedFlagsPage.js]: Being ultra picky, if we're not using this, can we remove it please?

File: src/Api/Controller/ListFlagsController.php [hunk 3]
```diff
@@ -47,11 +58,22 @@ protected function data(ServerRequestInterface $request, Document $document)
         $actor->read_flags_at = time();
```
Comment (imorland) [inline src/Api/Controller/ListFlagsController.php]: I don't think we should update this timestamp right here, if the call is coming from the dismissed UI. Perhaps this should be wrapped in the check for `!onlyDismissed`?

File: js/src/admin/components/DismissedFlagsTable.js (hunk 1 above)
Comment (harrygiffgaff) [reply js/src/admin/components/DismissedFlagsTable.js]: Added dismissed by column in table

File: src/Api/Controller/ListFlagsController.php (hunk 3 above)
Comment (harrygiffgaff) [reply src/Api/Controller/ListFlagsController.php]: Timestamp wrapped in onlyDismissed==false check

File: extend.php [hunk 4]
```diff
@@ -22,7 +22,10 @@
         ->css(__DIR__.'/less/forum.less'),
 
     (new Extend\Frontend('admin'))
-        ->js(__DIR__.'/js/dist/admin.js'),
+        ->js(__DIR__.'/js/dist/admin.js')
+        ->css(__DIR__.'/less/admin.less'),
+
+    new Extend\Locales(__DIR__.'/resources/locale'),
```
Comment (imorland) [inline extend.php]: This should also be removed, as part of the supporting PR to https://github.com/flarum/lang-english

Comment (andrewdyer) [reply extend.php]: https://github.com/flarum/lang-english/pull/160

**Pull request merged**